contents/docs/product-analytics/correlation.mdx

```diff
@@ -10,52 +10,40 @@ availability:
+Funnels have the ability to directly compare conversion rates of users for a specific property via the breakdown feature, but this requires you to know which property to investigate. If you're not certain, correlation analysis highlights significant events or properties for you. 
 
-Given a funnel, PostHog will calculate the conversion rate for different slices (either by events performed, properties they have or properties for the events they performed) and surface relevant signals. Below is an example of what correlation analysis might look like:
+As an example, the below insight shows that desktop users were `1.3x more likely` to convert than mobile or tablet users. Similarly, we also seee that users who completed the `uploaded_cv_for_review` event were `8.0x more likely` to convert.
 
-![Correlation analysis](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/docs/user-guides/funnels/funnel-correlation-analysis.png)
+<ProductScreenshot
+    imageLight = {"https://res.cloudinary.com/dmukukwp6/image/upload/v1716387676/posthog.com/contents/Screenshot_2024-05-22_at_3.20.17_PM.png"} 
+    imageDark = {"https://res.cloudinary.com/dmukukwp6/image/upload/v1716387676/posthog.com/contents/Screenshot_2024-05-22_at_3.20.33_PM.png"}
+    classes="rounded"
+    alt= "Example of a correlation analysis insight"
+/>
 
-## Using correlation analysis
+It's important to remember that the accuracy any correlation is dependent on the sample size. If you've only got a handful of users for a specific correlation, then it's likely that it will *not* be useful.
 
-Suppose we have a funnel and we want to optimize it. The first thing we might want to do is identify if there is anything special about user that make them more or less likely to convert. With correlation analysis we can automatically identify which user properties are significant:
+## How to create a correlation analysis
 
-![Person property correlations](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/docs/user-guides/funnels/funnel-correlation-user-properties.png)
+In any funnel insight, scroll to the **Correlation analysis** section and click the **Load results** or **Select properties** buttons.
 
-Above we can see that the conversion rate of people using Android and iOS are **4.1x** and **3.1x times less likely** to convert in our example funnel. However, we also see other properties that may be of interest, for instance the "Country
-Name". This gives us some threads to pull on, perhaps we might look into internationalization, or ensure the app loads well in different regions.
+Once you've created a correlation, you can also create a [cohort](/docs/data/cohorts) of users in an analysis by clicking the numbers in the **Completed** or **Dropoff** columns and then the **Save as cohort** button in the modal.
 
-Equally, we can see specific events that are correlated:
+## Excluding events or properties
```

Comment:
Should we also mention "correlation details" here?

![CleanShot 2024-05-22 at 11 44 43@2x](https://github.com/PostHog/posthog.com/assets/34755028/ea496b63-dd39-43ff-8018-67965416d196)

Reply:
I thought about it, but I think the feature is mostly self-explanatory. I dont think we need to documents every single bit of functionality